rasa/nlu/classifiers/embedding_intent_classifier.py

```diff
@@ -909,16 +875,26 @@ def __init__(
 
         # tf training
         self._optimizer = tf.keras.optimizers.Adam(config[LEARNING_RATE])
-        self.entity_f1 = tfa.metrics.F1Score(
-            num_classes=self._num_tags - 1,  # `0` prediction is not a prediction
-            average="micro",
-        )
+        self.intent_acc = tf.keras.metrics.Mean(name="i_acc")
+        self.intent_loss = tf.keras.metrics.Mean(name="i_loss")
+        self.mask_loss = tf.keras.metrics.Mean(name="m_loss")
+        self.mask_acc = tf.keras.metrics.Mean(name="m_acc")
+        self.entity_loss = tf.keras.metrics.Mean(name="e_loss")
+        self.entity_f1 = tf.keras.metrics.Mean(name="e_f1")
```

Comment:
so layers can be part of the dict but metrics not? Good job tensorflow, very intuitive 🙈

Reply:
yeah, if you want to collect the metrics afterwards, they need to be class variables...